Corona-Warn-App/src/main/res/values-en/legal_strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As this texts are under our control and english is our fallback language values/legal_strings.xml has to be updated, if english is selected and no dedicated translated strings are present the fallback strings will be used. This means one file we don't have to keep update. So please move this over to values/legal_strings.xml